### PR TITLE
Feature flair system + flair to wall feed

### PR DESCRIPTION
**Pull request by Sanjay-B**
Adds flair to wall feed + a system to expand flairs.

### CODE REVIEW

File: core/dashboard/pages/home.lua [hunk 1]
```diff
@@ -1,3 +1,77 @@
+-- Copyright 2020- Teverse.com
+-- Used to display the home screen of the teverse application
+
+local globals = require("tevgit:workshop/library/globals.lua") -- globals; variables or instances that can be shared between files
+
+local function createFlair(parent)
+    local username = parent:child("username").text
+    teverse.http:get("https://teverse.com/api/users/fromUsername/"..username, {
```
Comment (teverse) [inline core/dashboard/pages/home.lua:8]: Unnecessary call to API? Previously the feed endpoint is used in this file, this api call already returns the user's membership.

E.g.
`v.postedBy.beta`
`v.postedBy.membership`

Comment (Sanjay-B) [reply core/dashboard/pages/home.lua]: Has been corrected.

File: workshop/library/ui/controllers/sidebarInterface.lua [hunk 2]
```diff
@@ -35,15 +35,24 @@ return {
             position = guiCoord(0, 0, 0.24, 0),
             backgroundColour = globals.defaultColours.secondary,
         })
+<<<<<<< HEAD
```
Comment (teverse) [inline workshop/library/ui/controllers/sidebarInterface.lua:38]: This looks like something went wrong when resolving conflicts

File: workshop/library/ui/components/toolTip.lua [hunk 3]
```diff
@@ -28,7 +28,11 @@ return {
 
         if orientation == "vertical" then -- If orientation is specified to "vertical"
             local container = teverse.construct("guiFrame", {
+<<<<<<< HEAD
```
Comment (teverse) [inline workshop/library/ui/components/toolTip.lua:31]: bad merge here too?

File: workshop/library/ui/controllers/topbarInterface.lua [hunk 4]
```diff
@@ -33,7 +33,11 @@ return {
             backgroundColour = globals.defaultColours.white,
         })
 
+<<<<<<< HEAD
```
Comment (teverse) [inline workshop/library/ui/controllers/topbarInterface.lua:36]: bad merge here too?

File: workshop/main.lua [hunk 5]
```diff
@@ -60,8 +60,13 @@ return function(dev)
 
     -- If initialize phase fails, prompt to the error screen
     if (not success) then
+<<<<<<< HEAD
```
Comment (teverse) [inline workshop/main.lua:63]: bad merge here too?

Comment (Sanjay-B) [reply workshop/main.lua]: Fixed as of https://github.com/teverse/teverse/pull/72/commits/726cc11e45f894a44170bc45cd05f5ae415c7820.

File: workshop/library/ui/controllers/topbarInterface.lua (hunk 4 above)
Comment (Sanjay-B) [reply workshop/library/ui/controllers/topbarInterface.lua]: Fixed as of https://github.com/teverse/teverse/pull/72/commits/726cc11e45f894a44170bc45cd05f5ae415c7820.

File: workshop/library/ui/components/toolTip.lua (hunk 3 above)
Comment (Sanjay-B) [reply workshop/library/ui/components/toolTip.lua:31]: Fixed as of https://github.com/teverse/teverse/pull/72/commits/726cc11e45f894a44170bc45cd05f5ae415c7820.

File: workshop/library/ui/controllers/sidebarInterface.lua (hunk 2 above)
Comment (Sanjay-B) [reply workshop/library/ui/controllers/sidebarInterface.lua]: Fixed as of https://github.com/teverse/teverse/pull/72/commits/726cc11e45f894a44170bc45cd05f5ae415c7820.

**Comment by Sanjay-B** [comment]
It should be fine now.

**Pull request merged**